core/src/test/java/org/apache/accumulo/core/conf/PropertyTest.java

```diff
@@ -88,13 +88,28 @@ public void testPorts() {
     for (Property prop : Property.values()) {
       if (prop.getType().equals(PropertyType.PORT)) {
         int port = Integer.parseInt(prop.getDefaultValue());
+        assertTrue(Property.isValidProperty(prop.getKey(), Integer.toString(port)));
         assertFalse(usedPorts.contains(port), "Port already in use: " + port);
         usedPorts.add(port);
         assertTrue(port > 1023 && port < 65536, "Port out of range of valid ports: " + port);
       }
     }
   }
 
+  @Test
+  public void testBooleans() {
+    for (Property prop : Property.values()) {
```

Comment:
Is this loop necessary?  The test would be valid with any single boolean property.  Also, the test in ShellIT that you added is more comprehensive, testing each type - would it be better to test those conditions here?  Not sure if it needs to be in both places, but unit testing would be faster than needing to run an IT.

Reply:
Replaced `testBooleans` with `testPropertyValidation` that runs through the property possibilities and attempts invalid types. 

I left the ShellIT alone since it's a client test of the RPC calls. If the shell is ever spun out of this codebase it would remain valuable.